Conversions/DateToDay.js

```diff
@@ -40,10 +40,21 @@ const DateToDay = (date) => {
   const yearDigit = (year % 100)
   const century = Math.floor(year / 100)
 
-  // Apply the algorithm shown above
+  /**
```

Comment:
Don't copy verbatim from Wikipedia.

Reply:
I have tried to explain the modulo division, please check and let me know if it makes sense.
Also, I have updated the commit message as `comment update` instead doc (_which was incorrect in this case_).